src/ui/public/vislib/vis.js

```diff
@@ -25,12 +25,12 @@ export default function VisFactory(Private) {
    * @param config {Object} Parameters that define the chart type and chart options
    */
   class Vis extends Events {
-    constructor($el, config) {
+    constructor($el, visConfigArgs) {
       super(arguments);
       this.el = $el.get ? $el.get(0) : $el;
       this.binder = new Binder();
-      this._attr = config;
-      this._attr.el = this.el;
+      this.visConfigArgs = visConfigArgs;
```

Comment:
Is there anything in `Vis` that actually needs the `visConfigArgs`? Could we instantiate `VisConfig` first and then get the `el` from it?